RecoTauTag/RecoTau/interface/TauTagTools.h

```diff
@@ -0,0 +1,20 @@
+#ifndef RecoTauTag_RecoTau_TauTagTools_h
+#define RecoTauTag_RecoTau_TauTagTools_h
+
+#include "DataFormats/Candidate/interface/Candidate.h"
+#include "DataFormats/Candidate/interface/CandidateFwd.h"
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/VertexReco/interface/Vertex.h"
+#include "DataFormats/VertexReco/interface/VertexFwd.h"
+
+namespace TauTagTools{
```

Comment:
namespaces should be all in lower case 
http://cms-sw.github.io/cms_coding_rules.html#2--naming-rules-1

Reply:
It was mostly a simple copy from the removed TauTagTools folder (removing unneeded code in this file), but I fixed it nonetheless.